blueprints/banking_assistant/__init__.py

```diff
@@ -0,0 +1,498 @@
+}
+
+# Dialogue state handlers
+
+
+@app.handle(intent="greet")
+def greet(request, responder):
+    global user
+    if not user:
+        user = User(request)
+    responder.slots["name"] = user.get("first_name")
+    replies = [
+        "Hi {name}, how can I help with your banking tasks? You can try transferring "
+        "money between accounts or paying a bill.",
+        "Hello {name}, thanks for choosing MindMeld Bank. You can do things "
+        "like: ask for your routing number, order "
+        "checks, and check bill due dates.",
+        "Welome to MindMeld Bank {name}, How can I help with your banking tasks? "
+        "You can try reporting a fraud charge or a lost credit card.",
+        "Thanks for using MindMeld Bank {name}! What would you like to do today? "
+        "A few things I can help with are, "
+        "checking balances, paying off your credit card, and setting up a new card.",
+    ]
+    responder.reply(replies)
+
+
+@app.handle(intent="exit")
+def exit(request, responder):
+    responder.reply(["Bye", "Goodbye", "Have a nice day."])
+
+
+@app.handle(intent="help")
+def help(request, responder):
+    responder.reply(
+        [
+            "A few things I can help you with are: checking balances,"
+            " paying off your credit card, and setting up a new card.",
+            "Looks like you need help. Have you tried: asking for your routing"
+            " number, ordering checks, and checking bill due dates.",
+            "You can try reporting a fraud charge or a lost credit card.",
+            "Need some suggestions? Try ordering some "
+            "checks by saying 'order checks', or check your balance for savings,"
+            " checking, or credit account by saying 'check balance' and then the account name",
+            "A few of things you can ask me are: 'how to apply for a loan', 'what is my routing "
+            "number', and 'can you pay my credit card bill'",
+            "Try asking about card activation or reporting a stolen card",
+            "A few things I can help with are: reporting a fraudulent charge,"
+            " paying off your credit card, and resetting your PIN.",
+            "Have you tried asking for your routing number yet? You can do that by saying 'what "
+            "is my routing number'.",
+            "Have you tried making a transfer? You can do that by saying 'transfer money'.",
+            "Need some suggestions? You can ask me something like 'lost my PIN', 'card is "
+            "stolen', or 'setup a new card'.",
+        ]
+    )
+
+
+@app.handle(intent="lost_creditcard")
+def faq_lost_creditcard_handler(request, responder):
+    global user
+    if not user:
+        user = User(request)
+    responder.slots["email"] = user.get("email")
+    replies = [
+        "I've noted that your card may have been lost or stolen. Please follow up "
+        "immediately by calling 1-800-432-3117 or clicking the link in the email sent to {email}."
```

Comment:
We shouldn't use the real Chase Freedom number here.
```suggestion
        "immediately by calling 1-800-555-7456 or clicking the link in the email sent to {email}."
```